src/client/lib/CliAcquisition.ts

```diff
@@ -61,6 +63,7 @@ export class CliAcquisition implements IDisposable {
     async installCli(pathToNupkg: string): Promise<string> {
         const pacToolsPath = path.join(this._cliPath, 'tools');
         if (this.isCliExpectedVersion()) {
+            this._context.telemetry.sendTelemetryEvent('PacCliAcquired', { cliVersion: this.cliVersion, installState: 'AlreadyInstalled' });
```

Comment:
that is likely too noisy: this code path is walked on every extension activation, e.g. when opening or reloading a VSCode window.  -> remove

Reply:
fair enough. I'll cleanup the other event below then too.